src/styles/table.scss

```diff
@@ -11,6 +11,9 @@
 */
 $block: #{$fd-namespace}-table;
 
+$fd-table-border: 0.0625rem solid #e4e4e4;
```

Comment:
You don't need this part, it's a fallback generated with a script to make sure variables work in all browsers. 
All you need is:
```
$fd-table-border: var(--sapList_BorderWidth) solid var(--sapList_BorderColor);
```

Reply:
done